### PR TITLE
Bugfix/pod configuration

### LINKED ISSUE #19: [IOS]info ld: library not found for -lRNSelectableText
I have do this:
```
yarn add @astrocoders/react-native-selectable-text
react-native link @astrocoders/react-native-selectable-text
```

when I run `react-native run-android` that is ok.
But when I run `react-native run-ios` have error:
```
info ld: library not found for -lRNSelectableText
```
dependencies:
```
"@astrocoders/react-native-selectable-text": "^1.2.1",
"react-native": "0.59.9",
```

### PR DESCRIPTION
**Pull request by tobiaseisenschenk**
I was only able to use this library with the shown modifications. Since [Pods will become more important to manage native iOS dependencies](https://github.com/facebook/react-native/releases/tag/v0.60.0-rc.3), I adapted the readme and created this PR. Happy to discuss any changes!

---

fixes #19

### CODE REVIEW

**Comment by georgelima** [comment]
Thanks @tobiaseisenschenk! I’ll try it soon @guilhermedecampo

**Comment by fakenickels** [comment]
Nice!

File: ios/RNSelectableTextView.m [hunk 1]
```diff
@@ -1,8 +1,22 @@
-
+#if __has_include(<RCTText/RCTTextSelection.h>)
```
Comment (fakenickels) [inline ios/RNSelectableTextView.m]: was this file generated? if so, how did you do it?

Comment (tobiaseisenschenk) [reply ios/RNSelectableTextView.m]: No it was not auto generated, I just had a look at how e.g. react-native-sentry solves this issue and copied the pattern.

**Comment by tobiaseisenschenk** [comment]
Can I help further with resolving the conflicts?

**Comment by medson10** [comment]
Hi @tobiaseisenschenk, thanks for your contribution

I tested in a real project and it's still showing the error

image

I installed the new version, added the pod and installed the new pods, are there any steps I'm missing?

**Comment by tobiaseisenschenk** [comment]
> Hi @tobiaseisenschenk, thanks for your contribution
> 
> I tested in a real project and it's still showing the error
> 
> image
> 
> I installed the new version, added the pod and installed the new pods, are there any steps I'm missing?

Are you already managing your RN dependencies in your podfile? [This article explains the general setup](https://sandstorm.de/de/blog/post/react-native-managing-native-dependencies-using-xcode-and-cocoapods.html). Do you have libPods added to the target's "linked frameworks and libraries" in XCode?

**Comment by medson10** [comment]
> > Hi @tobiaseisenschenk, thanks for your contribution
> > I tested in a real project and it's still showing the error
> > image
> > I installed the new version, added the pod and installed the new pods, are there any steps I'm missing?
> 
> Are you already managing your RN dependencies in your podfile? [This article explains the general setup](https://sandstorm.de/de/blog/post/react-native-managing-native-dependencies-using-xcode-and-cocoapods.html). Do you have libPods added to the target's "linked frameworks and libraries" in XCode?

Yes, I have other dependencies using pods, they are working fine

**Comment by tobiaseisenschenk** [comment]
> Yes, I have other dependencies using pods, they are working fine
For me `RCTBaseTextInputView.h` points to this path in xcode:
Screen Shot 2019-07-16 at 3 18 04 PM
Do you also have this binary linked under Libraries?

**Comment by medson10** [comment]
> > Yes, I have other dependencies using pods, they are working fine
> 
> For me `RCTBaseTextInputView.h` points to this path in xcode:
> Screen Shot 2019-07-16 at 3 18 04 PM
> Do you also have this binary linked under Libraries?

image

yes, if I remove the selectable text from the linked libraries and build, then add again and build, it works

**Comment by tobiaseisenschenk** [comment]
> yes, if I remove the selectable text from the linked libraries and build, then add again and build, it works
>
Ah ok i think the issue is that you have selectableText as linked library at all! I have libPods as first item in "link binary with libraries" since any library that you install through pods comes through the libPods item and therefore doesn't have to be linked additionally anymore.

**Comment by guilhermedecampo** [comment]
Would be nice to have something in the README about this possible error!

**Comment by medson10** [comment]
> > yes, if I remove the selectable text from the linked libraries and build, then add again and build, it works
> 
> Ah ok i think the issue is that you have selectableText as linked library at all! I have libPods as first item in "link binary with libraries" since any library that you install through pods comes through the libPods item and therefore doesn't have to be linked additionally anymore.

Indeed removed the link and is working now, sorry my bad, approved and after fix the conflicts can merge

**Comment by fakenickels** [comment]
nice good catch!

**Comment by fakenickels** [comment]
Thanks @tobiaseisenschenk !

**Pull request merged**